Tests/UnitTests/Purchasing/StoreKitWrapperTests.swift

```diff
@@ -247,4 +263,18 @@ class StoreKitWrapperTests: XCTestCase, StoreKitWrapperDelegate {
         expect(payment2.simulatesAskToBuyInSandbox) == true
     }
 
+#if os(iOS) || targetEnvironment(macCatalyst)
+    func testShouldShowPriceConsentWiredUp() throws {
+        guard #available(iOS 13.4, macCatalyst 13.4, *) else {
+            throw XCTSkip()
+        }
+        expect(self.shouldShowPriceConsent) == true
+
+        self.shouldShowPriceConsent = false
+
+        let consentStatuses = self.paymentQueue.simulatePaymentQueueShouldShowPriceConsent()
```

Comment:
maybe we could (as a separate thing) expose this too? like we expose [`simulatesAskToBuyInSandbox`](https://revenuecat.github.io/purchases-ios-docs/4.2.1/documentation/revenuecat/purchases/simulatesasktobuyinsandbox)

Reply:
added https://revenuecats.atlassian.net/browse/CF-599